epicbox-hyperskill/python/checker/check.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-cd /sandbox
-python tests.py --inside_docker > stdout.txt 2> stderr.txt
-echo $? > code.txt
+cd /sandbox/test
+python tests.py --inside_docker > ../stdout.txt 2> ../stderr.txt
+echo $? > ../code.txt
```

Comment:
The working directory must not be changed (/sandbox).
```bash
python -m unittest discover -s test > stdout.txt 2> stderr.txt
```
This is about, you need to check if it really works.
https://docs.python.org/3/library/unittest.html

You also need to edit the hs-test-python library code so that the `inside_docker` parameter is passed through an environment variable.

https://docs.docker.com/engine/reference/builder/#env
https://docs.python.org/3/library/os.html#os.environ